eng/testing/tests.mobile.targets

```diff
@@ -82,12 +85,16 @@
       <AndroidAbi Condition="'$(TargetArchitecture)' == 'x86'">x86</AndroidAbi>
 
       <MainLibraryFileName Condition="'$(MainLibraryFileName)' == ''">AndroidTestRunner.dll</MainLibraryFileName>
+      <RuntimeConfigFilePath>$(FunctionalTestsDir)/Android/Device_Emulator/JIT/runtimeconfig.template.json</RuntimeConfigFilePath>
```

Comment:
I'm not sure what this is doing.

1. Is it adding the runtime config template from the functional test to every testsuite?  That seems wrong.
2. Why is it unsing the `.template.json` file from the source directory?  it should be using the generated `runtimeconfig.json` from the output directory (ie from "bin/Debug/.../runtimeconfig.json") or maybe from the publish directory (I'm not sure) - there is probably some property that is set to the path to that file.  I'm not sure.
3. It probably doesn't need to be part of the `BundleTestAndroidApp` target.  It can be its own target that the `PublishTestAsSelfContained` task can depend on.
4. I don't like the "output.pefile" filename.  (a) it's not a PE file anymore, (b) "output" doesn't really mean anything.  How about "runtimeconfig.bin" or something?

Reply:
It was just a handy variable for me to find the `runtimeconfig.template.json` file. Since I am supposed to parse the generated `*.runtimeconfig.json` file. This var is no longer needed.